src/libmongoc/tests/unified/operation.c

```diff
@@ -2485,6 +2605,9 @@ operation_run (test_t *test, bson_t *op_bson, bson_error_t *error)
          }
          if (!result_check (result,
                             test->entity_map,
+                            op->ignore_result_and_error
+                               ? *op->ignore_result_and_error
+                               : false,
```

Comment:
Just to confirm, `result_check` is used for _both_ result _and_ error checks, correct? So this instructs it to be a no-op? Any reason you don't just avoid calling `result_check` altogether if `ignore_result_and_error` is set? Do we rely on `result_check` to catch other test format errors that aren't considered `expectError` (e.g. bad syntax)?

Reply:
I've changed this to not call `result_check`. There was no reason for doing it that way except that I didn't think of any other way...